beancount/parser/booking_full.py

```diff
@@ -195,24 +195,48 @@ def _book(entries, options_map, methods):
 
```

Comment:
Did you measure the performance diff?
I'm seeing little tiny speedup - strangely, which is concerning. 
Could be statistical noise.